src/views/dashboard/Support.vue

```diff
@@ -142,21 +223,14 @@ export default {
 </script>
 
 <style scoped>
-.productList {
-  @apply space-y-2;
+.products {
+  @apply flex space-x-2;
```

Comment:
I think the standard spacing is `space-x-4` for the panels. 
I imagine you just haven't gotten round to it, but this will be more like `@apply flex flex-col space-y-4 sm:flex-row sm:space-y-0 sm:space-x-4`

Reply:
I will come back to this in a future PR - need to get these changes merged.